lib/src/protocols/websocket_io.dart

```diff
@@ -81,7 +85,7 @@ class KuzzleWebSocket extends KuzzleProtocol {
     super.close();
 
     removeAllListeners();
-    stopRetryingToConnect = true;
+    _stopRetryingToConnect = true;
```

Comment:
On close this variable is set to true to cancel future reconnection attempt but this is never set back to false, meaning that if you do close the connection at one point, then do a connect, the auto reconnect wont work because of `_stopRetryingToConnect=true`

you should reset it back to false at some point